[2024-08-03T16:59:26.011+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: excel_to_oracle_dag.load_excel_to_oracle manual__2024-08-03T16:59:24.815210+00:00 [queued]>
[2024-08-03T16:59:26.021+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: excel_to_oracle_dag.load_excel_to_oracle manual__2024-08-03T16:59:24.815210+00:00 [queued]>
[2024-08-03T16:59:26.022+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-08-03T16:59:26.023+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2024-08-03T16:59:26.023+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-08-03T16:59:26.037+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): load_excel_to_oracle> on 2024-08-03 16:59:24.815210+00:00
[2024-08-03T16:59:26.044+0000] {standard_task_runner.py:55} INFO - Started process 1216 to run task
[2024-08-03T16:59:26.047+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'excel_to_oracle_dag', 'load_excel_to_oracle', 'manual__2024-08-03T16:59:24.815210+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/excel_to_oracle_dag.py', '--cfg-path', '/tmp/tmpkddu7jk6']
[2024-08-03T16:59:26.051+0000] {standard_task_runner.py:83} INFO - Job 25: Subtask load_excel_to_oracle
[2024-08-03T16:59:26.108+0000] {task_command.py:376} INFO - Running <TaskInstance: excel_to_oracle_dag.load_excel_to_oracle manual__2024-08-03T16:59:24.815210+00:00 [running]> on host 92fc8587d3bd
[2024-08-03T16:59:26.175+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=excel_to_oracle_dag
AIRFLOW_CTX_TASK_ID=load_excel_to_oracle
AIRFLOW_CTX_EXECUTION_DATE=2024-08-03T16:59:24.815210+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-03T16:59:24.815210+00:00
[2024-08-03T16:59:26.185+0000] {base.py:71} INFO - Using connection ID 'Connection bai test' for task execution.
[2024-08-03T16:59:26.186+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/excel_to_oracle_dag.py", line 44, in load_excel_to_oracle
    connection = cx_Oracle.connect(user=conn.login, password=conn.password, dsn=dsn)
cx_Oracle.DatabaseError: DPI-1047: Cannot locate a 64-bit Oracle Client library: "libclntsh.so: cannot open shared object file: No such file or directory". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help
[2024-08-03T16:59:26.196+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=excel_to_oracle_dag, task_id=load_excel_to_oracle, execution_date=20240803T165924, start_date=20240803T165926, end_date=20240803T165926
[2024-08-03T16:59:26.208+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 25 for task load_excel_to_oracle (DPI-1047: Cannot locate a 64-bit Oracle Client library: "libclntsh.so: cannot open shared object file: No such file or directory". See https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html for help; 1216)
[2024-08-03T16:59:26.218+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2024-08-03T16:59:26.239+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
